src/App.js

```diff
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import LogIn from './pages/logins/LogIn';
+import SignUp from './pages/registration/Register';
+import Home from './pages/Home';
+import MentorsPage from './pages/mentors/MentorsPage';
+import MentorsDetails from './pages/mentors/MentorsDetails';
+import ReservationForm from './pages/reservations/ReservationForm';
+import ReservationsPage from './pages/reservations/ReservationsPage';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latest standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.